staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -164,6 +165,7 @@ func NewCmdDiff(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 	usage := "contains the configuration to diff"
 	cmd.Flags().StringArray("prune-allowlist", []string{}, "Overwrite the default whitelist with <group/version/kind> for --prune")
 	cmd.Flags().Bool("prune", false, "Include resources that would be deleted by pruning. Can be used with -l and default shows all resources would be pruned")
+	cmd.Flags().BoolVar(&options.IncludeManagedFields, "include-managed-fields", options.IncludeManagedFields, "If true, include managed fields in the diff.")
```

Comment:
As discussed during sig-cli call, change this to `--show-managed-fields` to be consistent with `kubectl get`.

Reply:
Flag name has been changed